### PR TITLE
Adding Notifications for Remove Books Requests 

### PR DESCRIPTION
**Pull request by MuneebNasir**
1. Updated the Remove Requests 
2. Modified the Notifications Message (Books Controller)

### CODE REVIEW

**Review by YoussefSaghbini — APPROVED** [review]
Thank you for the adjustment

File: src/main/js/pages/BookInterface/RemoveBookForm.js [hunk 1]
```diff
@@ -27,6 +28,12 @@ let LayoutTextFields = () => {
             url: `/api/removeBook?id=${id}`,
         }).then(res => {
             console.log(res);
```
Comment (BoucherR) [inline src/main/js/pages/BookInterface/RemoveBookForm.js:30]: Console.log accidentally left in

**Review by BoucherR — COMMENTED** [review]
Good stuff - we just need to remove that console.log that was left in, and LGTM!

**Review by vanarman — APPROVED** [review]
Looks great to me 👍

**Review by decarbonite — APPROVED** [review]
Good work

**Pull request merged**